quinn/__init__.py

```diff
@@ -5,3 +5,4 @@
 from .functions import *
 from .scala_to_pyspark import ScalaToPyspark
 from .transformations import *
+from .append_if_schema_identical import *
```

Comment:
We [decided](https://github.com/MrPowers/quinn/issues/75) that `import *` is a bad way so please use here explicit import of only single public function.

Reply:
I will make the changes as suggested.